packages/test-base/src/commonTest/kotlin/io/realm/kotlin/test/common/RealmConfigurationTests.kt

```diff
@@ -246,7 +246,7 @@ class RealmConfigurationTests {
     fun defaultLogLevel() {
         val config: RealmConfiguration = RealmConfiguration.Builder(schema = setOf(Sample::class))
             .build()
-        assertEquals(LogLevel.WARN, config.log.level)
+        assertEquals(LogLevel.WARN, RealmLog.level)
```

Comment:
Don't we still need to assert that `config.log.level` is correct? Maybe assert both.